{"uid":"256f55266db6b9e5","name":"test_address_prefix[Test for Polkadot Asset Hub]","fullName":"tests.test_network_parameters.TestNetworkPrefix#test_address_prefix","historyId":"694c7734c23968ac9df92475cad968b1","time":{"start":1738487138556,"stop":1738487138556,"duration":0},"status":"broken","statusMessage":"tests.conftest.FixtureTimeoutError: Timeout 15 when initializing properties: Polkadot Asset Hub","statusTrace":"timeout = 15\nfunction = <bound method Chain.init_properties of <scripts.utils.chain_model.Chain object at 0x7f744233a720>>\nargs = ()\nerror_message = 'Timeout 15 when initializing properties: Polkadot Asset Hub'\n\n    def execute_with_timeout(timeout: int, function, args: tuple, error_message: str):\n        try:\n>           return func_timeout(timeout=timeout, func=function, args=args)\n\ntests/conftest.py:32: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ntimeout = 15\nfunc = <bound method Chain.init_properties of <scripts.utils.chain_model.Chain object at 0x7f744233a720>>\nargs = (), kwargs = {}\n\n    def func_timeout(timeout, func, args=(), kwargs=None):\n        '''\n            func_timeout - Runs the given function for up to #timeout# seconds.\n    \n            Raises any exceptions #func# would raise, returns what #func# would return (unless timeout is exceeded), in which case it raises FunctionTimedOut\n    \n            @param timeout <float> - Maximum number of seconds to run #func# before terminating\n    \n            @param func <function> - The function to call\n    \n            @param args    <tuple> - Any ordered arguments to pass to the function\n    \n            @param kwargs  <dict/None> - Keyword arguments to pass to the function.\n    \n    \n            @raises - FunctionTimedOut if #timeout# is exceeded, otherwise anything #func# could raise will be raised\n    \n            If the timeout is exceeded, FunctionTimedOut will be raised within the context of the called function every two seconds until it terminates,\n            but will not block the calling thread (a new thread will be created to perform the join). If possible, you should try/except FunctionTimedOut\n            to return cleanly, but in most cases it will 'just work'.\n    \n            @return - The return value that #func# gives\n        '''\n    \n        if not kwargs:\n            kwargs = {}\n        if not args:\n            args = ()\n    \n        ret = []\n        exception = []\n        isStopped = False\n    \n        def funcwrap(args2, kwargs2):\n            try:\n                ret.append( func(*args2, **kwargs2) )\n            except FunctionTimedOut:\n                # Don't print traceback to stderr if we time out\n                pass\n            except Exception as e:\n                exc_info = sys.exc_info()\n                if isStopped is False:\n                    # Assemble the alternate traceback, excluding this function\n                    #  from the trace (by going to next frame)\n                    # Pytohn3 reads native from __traceback__,\n                    # python2 has a different form for \"raise\"\n                    e.__traceback__ = exc_info[2].tb_next\n                    exception.append( e )\n    \n        thread = StoppableThread(target=funcwrap, args=(args, kwargs))\n        thread.daemon = True\n    \n        thread.start()\n        thread.join(timeout)\n    \n        stopException = None\n        if thread.is_alive():\n            isStopped = True\n    \n            class FunctionTimedOutTempType(FunctionTimedOut):\n                def __init__(self):\n                    return FunctionTimedOut.__init__(self, '', timeout, func, args, kwargs)\n    \n            FunctionTimedOutTemp = type('FunctionTimedOut' + str( hash( \"%d_%d_%d_%d\" %(id(timeout), id(func), id(args), id(kwargs))) ), FunctionTimedOutTempType.__bases__, dict(FunctionTimedOutTempType.__dict__))\n    \n            stopException = FunctionTimedOutTemp\n            thread._stopThread(stopException)\n            thread.join(min(.1, timeout / 50.0))\n>           raise FunctionTimedOut('', timeout, func, args, kwargs)\nE           func_timeout.exceptions.FunctionTimedOut: Function init_properties (args=()) (kwargs={}) timed out after 15.000000 seconds.\n\n.venv/lib/python3.12/site-packages/func_timeout/dafunc.py:101: FunctionTimedOut\n\nDuring handling of the above exception, another exception occurred:\n\nrequest = <SubRequest 'chain_model' for <Function test_address_prefix[Test for Polkadot Asset Hub]>>\n\n    @pytest.fixture(scope=\"module\", params=get_substrate_chains(), ids=chain_names)\n    def chain_model(request) -> Chain:\n        chain = request.param\n        execute_with_timeout(\n            timeout=FIXTURE_TIMEOUT,\n            function=chain.create_connection,\n            args=(),\n            error_message=f'Timeout {FIXTURE_TIMEOUT} when creating connection: {chain.name}'\n        )\n>       execute_with_timeout(\n            timeout=FIXTURE_TIMEOUT,\n            function=chain.init_properties,\n            args=(),\n            error_message=f'Timeout {FIXTURE_TIMEOUT} when initializing properties: {chain.name}'\n        )\n\ntests/conftest.py:81: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ntimeout = 15\nfunction = <bound method Chain.init_properties of <scripts.utils.chain_model.Chain object at 0x7f744233a720>>\nargs = ()\nerror_message = 'Timeout 15 when initializing properties: Polkadot Asset Hub'\n\n    def execute_with_timeout(timeout: int, function, args: tuple, error_message: str):\n        try:\n            return func_timeout(timeout=timeout, func=function, args=args)\n        except FunctionTimedOut:\n            with allure.step(\"Function timed out\"):\n                allure.attach(name=\"Error\", body=error_message,\n                              attachment_type=allure.attachment_type.TEXT)\n>           raise FixtureTimeoutError(error_message)\nE           tests.conftest.FixtureTimeoutError: Timeout 15 when initializing properties: Polkadot Asset Hub\n\ntests/conftest.py:37: FixtureTimeoutError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"chain_model","time":{"start":1738487138557,"stop":1738487154785,"duration":16228},"status":"broken","statusMessage":"tests.conftest.FixtureTimeoutError: Timeout 15 when initializing properties: Polkadot Asset Hub\n","statusTrace":"  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1128, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 907, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 81, in chain_model\n    execute_with_timeout(\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 37, in execute_with_timeout\n    raise FixtureTimeoutError(error_message)\n","steps":[{"name":"Function timed out","time":{"start":1738487154785,"stop":1738487154785,"duration":0},"status":"passed","steps":[],"attachments":[{"uid":"d768a4392f3b19dc","name":"Error","source":"d768a4392f3b19dc.txt","type":"text/plain","size":59}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},{"name":"chain_model","time":{"start":1738487159856,"stop":1738487175972,"duration":16116},"status":"broken","statusMessage":"tests.conftest.FixtureTimeoutError: Timeout 15 when initializing properties: Polkadot Asset Hub\n","statusTrace":"  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1128, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 907, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 81, in chain_model\n    execute_with_timeout(\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 37, in execute_with_timeout\n    raise FixtureTimeoutError(error_message)\n","steps":[{"name":"Function timed out","time":{"start":1738487175971,"stop":1738487175972,"duration":1},"status":"passed","steps":[],"attachments":[{"uid":"da7e95470060fd61","name":"Error","source":"da7e95470060fd61.txt","type":"text/plain","size":59}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},{"name":"chain_model","time":{"start":1738487180993,"stop":1738487198005,"duration":17012},"status":"broken","statusMessage":"tests.conftest.FixtureTimeoutError: Timeout 15 when initializing properties: Polkadot Asset Hub\n","statusTrace":"  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1128, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/.venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 907, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 81, in chain_model\n    execute_with_timeout(\n  File \"/home/runner/work/test-runner/test-runner/nova-utils/tests/conftest.py\", line 37, in execute_with_timeout\n    raise FixtureTimeoutError(error_message)\n","steps":[{"name":"Function timed out","time":{"start":1738487198004,"stop":1738487198004,"duration":0},"status":"passed","steps":[],"attachments":[{"uid":"1d36f55039d70144","name":"Error","source":"1d36f55039d70144.txt","type":"text/plain","size":59}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_network_parameters"},{"name":"subSuite","value":"TestNetworkPrefix"},{"name":"host","value":"fv-az1267-952"},{"name":"thread","value":"5172-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_network_parameters"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"chain_model","value":"<scripts.utils.chain_model.Chain object at 0x7f744233a720>"}],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"256f55266db6b9e5.json","parameterValues":["<scripts.utils.chain_model.Chain object at 0x7f744233a720>"]}